browser-extension/chrome/popup/styles.css

```
@@ -0,0 +1,77 @@
+@tailwind base;
```

Biome (2.3.13)

[error] Unexpected unknown at-rule: tailwind 

tailwind is not a standard CSS at-rule, which may lead to unexpected styling results or failure to interpret the styles as intended.
See MDN web docs for a known list of at-rules.
To fix this issue, consider removing the unknown at-rule.

(lint/suspicious/noUnknownAtRules)